cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -132,7 +132,7 @@ object AnalyzerCommand : CommandWithHelp() {
         val ortResult = analyzer.analyze(absoluteProjectPath, packageManagers, packageCurationsFile,
                 repositoryConfigurationFile)
 
-        println("Found ${ortResult?.analyzer?.result?.projects.orEmpty().size} project(s) in total.")
+        println("Found ${ortResult.analyzer?.result?.projects.orEmpty().size} project(s) in total.")
```

Comment:
That change should be in the previous commit.

Reply:
Good spotting!